Frameworks/UIKit/UISegment.mm

```diff
@@ -36,53 +36,56 @@ + (instancetype)initialize {
 }
 
 - (instancetype)initWithCoder:(NSCoder*)coder {
-    id result = [super initWithCoder:coder];
-    id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
-    _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
+    if (self = [super initWithCoder:coder]) {
+        id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
+        _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
 
-    CGRect frame;
-    frame = [self bounds];
+        CGRect frame;
```

Comment:
`frame` isn't even used here.

---
In reply to: [89177950](https://github.com/Microsoft/WinObjC/pull/1441#discussion_r89177950) [](ancestors = 89177950)